api/base/exceptions.py

```diff
@@ -41,3 +41,8 @@ def json_api_exception_handler(exc, context):
 class Gone(APIException):
     status_code = status.HTTP_410_GONE
     default_detail = ('The requested resource is no longer available.')
+
+
+class InvalidFilter(ParseError):
```

Comment:
Yes, but those are named in turn after HTTP Errors. This is more akin to DRF classes like `ValidationError` or `ParseError` (which this notably subclasses).